services/src/atdd-staging/kilda.properties.example

```diff
@@ -10,4 +10,8 @@ floodlight.endpoint=http://localhost:8081
 floodlight.username=username
 floodlight.password=password
 
+elasticsearch.endpoint = http://elasticsearch.pendev:9200
```

Comment:
Let's use 'localhost' instead of 'elasticsearch.pendev' as it is done for other properties.

Reply:
sure